.bazelrc

```diff
@@ -26,6 +27,12 @@ build --host_cxxopt='-Wno-deprecated-declarations'
 # is our attempt to work around that.
 build --cxxopt='-Wno-unknown-warning-option'
 build --host_cxxopt='-Wno-unknown-warning-option'
+# The fuzztest library also has a conflict in one of its `.cc` files in
+# a macro. When https://github.com/google/fuzztest/issues/41 is fixed, we
+# should re-enable this warning.
+build --cxxopt='-Wno-macro-redefined'
+build --host_cxxopt='no-macro-redefined'
```

Comment:
typo `-Wno-macro-defined`

Reply:
No, I think this is right. The warning is complaining about a macro being defined twice with the contents of the definition differing. From the build error:

```
external/com_google_absl/absl/base/internal/thread_annotations.h:220:9: error: 'NO_THREAD_SAFETY_ANALYSIS' macro redefined [-Werror,-Wmacro-redefined]
```

Reply:
Oh, but I do need a -W there, that's probably what you meant.

Reply:
Fixed, good catch, thanks!